electricity/src/lib.rs

```diff
@@ -1,53 +1,150 @@
         },
         ShutDownParseData {
-            days_number: 0,
             page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-beograd/Dan_3_Iskljucenja.htm"),
             city: String::from("Belgrade")
         },
-        
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_0_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        },
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_1_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        },
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_2_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        },
+        ShutDownParseData {
+            page_url: String::from("https://elektrodistribucija.rs/planirana-iskljucenja-srbija/NoviSad_Dan_3_Iskljucenja.htm"),
+            city: String::from("Novi Sad")
+        }
     ];
 
-    for date in dates {
-        let html = fetch_page(&date.page_url).await?;
+    let mut table_rows: Vec<TableRowData> = vec![];
+
+    for input in dates {
+        let html = fetch_page(&input.page_url).await.unwrap();
+        let document = Html::parse_document(&html);
+        let table_selector = Selector::parse("table").unwrap();
+        let mut tables = document.select(&table_selector);
+        
+        let header_table = tables.nth(0).unwrap();
+        let header_selector = Selector::parse("tbody > tr > td > b").unwrap();
+        let header = header_table.select(&header_selector).next().unwrap();
+        let date = header.text().collect::<Vec<_>>().last().unwrap().to_string().split(" ").last().unwrap().to_string();
+
+        let table = tables.nth(0).unwrap();
+        let tr_selector = Selector::parse("tr").unwrap();
+        let td_selector = Selector::parse("td").unwrap();
+
+        for tr_element in table.select(&tr_selector).skip(1) {
+            let mut td_elements = tr_element.select(&td_selector);
+            let region: String = td_elements.nth(0).unwrap().text().collect::<Vec<_>>().into_iter().collect();
+            let time: String = td_elements.nth(0).unwrap().text().collect::<Vec<_>>().into_iter().collect();
+            let street: String = td_elements.nth(0).unwrap().text().collect::<Vec<_>>().into_iter().collect();
 
-        println!("{}", html);
+            println!("{}", region);
+            println!("{}", time);
+            println!("{}", street);
+
+            table_rows.push(
+                TableRowData { region, time, street, city: input.city.clone(), date: date.clone() }
+            )           
+        }
     }
 
-    Ok(())
+    let db_client = unsafe {
+        DATABASE.get_client().await
+    };
+
+    let _ = create_table(db_client, "electricity_failures", "id").await;
```

Comment:
I think, if we cannot create a table for some reason, it is better to propagate the error from the function.
Let's use crate `anyhow = { version = "1", features = [ "backtrace" ] }` for that for now.

Can we make the create_table return as a structure (let's say ElectroTable { ref db }). And the structure will have `add_failure(&self, row)` method? What do you think?